packages/protocol/contracts/layer1/preconf/iface/IOverseer.sol

```diff
@@ -9,48 +9,36 @@ interface IOverseer {
         uint48 unBlacklistedAt;
     }
 
-    /// @dev These delays prevent the lookahead from being messed up mid-epoch
-    struct Config {
-        // Delay after which a formerly unblacklisted operator can be blacklisted again
-        uint256 blacklistDelay;
-        // Delay after which a formerly blacklisted operator can be unblacklisted again
-        uint256 unblacklistDelay;
-    }
-
     // Blacklist events
     event Blacklisted(bytes32 indexed operatorRegistrationRoot, uint48 timestamp);
     event Unblacklisted(bytes32 indexed operatorRegistrationRoot, uint48 timestamp);
 
+    // Overseer management events
+    event OverseerUpdated(address indexed overseer, bool enabled);
+
     error BlacklistDelayNotMet();
+    error NotOverseer();
     error OperatorAlreadyBlacklisted();
     error OperatorNotBlacklisted();
     error UnblacklistDelayNotMet();
 
     /// @notice Blacklists a preconf operator for subjective faults
     /// @param _operatorRegistrationRoot registration root of the operator being blacklisted
-    /// @param _signatures signatures of the overseer signers
-    function blacklistOperator(
-        bytes32 _operatorRegistrationRoot,
-        bytes[] memory _signatures
-    )
-        external;
+    function blacklistOperator(bytes32 _operatorRegistrationRoot) external;
 
     /// @notice Removes an operator from the blacklist
     /// @param _operatorRegistrationRoot registration root of the operator to unblacklist
-    /// @param _signatures signatures of the overseer signers
-    function unblacklistOperator(
-        bytes32 _operatorRegistrationRoot,
-        bytes[] memory _signatures
-    )
-        external;
+    function unblacklistOperator(bytes32 _operatorRegistrationRoot) external;
+
+    /// @notice Sets or removes an overseer address
+    /// @dev Only the owner can add or remove overseers
+    /// @param _overseer The address to set as overseer
+    /// @param _enabled True to grant overseer role, false to revoke
+    function setOverseer(address _overseer, bool _enabled) external;
```

Comment:
Shall we add another function to allow an existing overseer to switch to another address? Otherwise, the owner will have to do it which will be much slower.